_posts/2025-04-27-20.0.0-release.md

```diff
@@ -0,0 +1,190 @@
+date: "2025-04-27 00:00:00"
+author: pmc
+categories: [release]
+---
+<!--
+{% comment %}
+Licensed to the Apache Software Foundation (ASF) under one or more
+contributor license agreements.  See the NOTICE file distributed with
+this work for additional information regarding copyright ownership.
+The ASF licenses this file to you under the Apache License, Version 2.0
+(the "License"); you may not use this file except in compliance with
+the License.  You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+{% endcomment %}
+-->
+
+The Apache Arrow team is pleased to announce the 20.0.0 release. This release
+covers over 2 months of development work and includes [**259 resolved
+issues**][1] on [**327 distinct commits**][2] from [**63 distinct
+contributors**][2]. See the [Install Page](https://arrow.apache.org/install/) to
+learn how to get the libraries for your platform.
+
+The release notes below are not exhaustive and only expose selected highlights
+of the release. Many other bugfixes and improvements have been made: we refer
+you to the [complete changelog][3].
+
+## Community
+
+Since the 19.0.0 release, Ed Seidl, Jean-Baptiste Onofré and Matthijs Brobbel
+have been invited to become committers. Bryce Mecum, Ian Cook, Jacob Wujciak-Jens 
+and Rok Mihevc have been invited to join the Project Management Committee (PMC).
+
+Thanks for your contributions and participation in the project!
+
+## C++ Notes
+
+### Compute
+
+
+### Acero
+
+
+### Filesystems
+
+### Flight RPC
+
+- The experimental Flight over UCX feature has been removed.
+([#43296](https://github.com/apache/arrow/issues/43296))
+
+
+### Parquet
+
+## C# Notes
+
+- Added support for the `Ordered` and `AppMetaData` fields to FlightInfo ([#45753](https://github.com/apache/arrow/pull/45753))
+- FlightClient can now be integrated with [Grpc.Net.ClientFactory](https://learn.microsoft.com/en-us/aspnet/core/grpc/clientfactory?view=aspnetcore-9.0) ([#45451](https://github.com/apache/arrow/issues/45451))
+
+## Java, Go, and Rust Notes
+
+The Java, Go, and Rust Go projects have moved to separate repositories outside
+the main Arrow [monorepo](https://github.com/apache/arrow).
+
+- For notes on the latest release of the [Java
+implementation](https://github.com/apache/arrow-java), see the latest [Arrow
+Java changelog][7].
+- For notes on the latest release of the [Rust
+  implementation](https://github.com/apache/arrow-rs) see the latest [Arrow Rust
+  changelog][5].
+- For notes on the latest release of the [Go
+implementation](https://github.com/apache/arrow-go), see the latest [Arrow Go
+changelog][6].
+
+## Linux Packaging Notes
+
```

Comment:
```suggestion

https://apache.jfrog.io/ is still available but https://packages.apache.org/ is preferred because the latter uses the apache.org domain.

```